### ISSUE TITLE
Could you also share the training code and main function?

### ISSUE THREAD

**Issue by Dyyyg** [heqin-zhu/BPfold#5]
Would it be possible to also release the training code, the main function , and the testing/evaluation code? I'd like to try training / fine-tuning on my own dataset to reproduce or extend your method.Even just the key parts (data loading, loss, training loop, evaluation) would be super helpful.Thanks again!

**Comment by heqin-zhu** [comment]
The evaluation code is available at [evaluation.py](https://github.com/heqin-zhu/BPfold/blob/master/src/BPfold/evaluate.py).

**Comment by Dyyyg** [comment]
Thanks again for your great work and for sharing it so generously.
